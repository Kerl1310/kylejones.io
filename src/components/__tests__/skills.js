import React from 'react';
import renderer from 'react-test-renderer';

import Skills from '../skills';

describe('Skills', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Skills
          title="Skills"
          skills={[
            {
              name: 'my-skill',
              level: 0,
            },
          ]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
