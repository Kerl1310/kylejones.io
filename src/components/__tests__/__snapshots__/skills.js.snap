// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills renders correctly 1`] = `
<div
  className="sc-bwzfXH jwfMIB"
>
  <h1>
    Skills
  </h1>
  <div
    className="sc-bdVaJa jQkEGY"
  >
    <label
      htmlFor="my-skill-bar"
    >
      my-skill
    </label>
    <div
      className="skill__bar"
      id="my-skill-bar"
    >
      <div
        className="skill__level"
      />
    </div>
  </div>
</div>
`;
